ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Stream.scala

```diff
@@ -38,16 +38,19 @@ private[h2] class H2Stream[F[_]: Concurrent](
     val id: Int,
     localSettings: H2Frame.Settings.ConnectionSettings,
     connectionType: H2Connection.ConnectionType,
-    val remoteSettings: F[H2Frame.Settings.ConnectionSettings],
+    remoteSettings: F[H2Frame.Settings.ConnectionSettings],
     val state: Ref[F, H2Stream.State[F]],
-    val hpack: Hpack[F],
-    val enqueue: cats.effect.std.Queue[F, Chunk[H2Frame]],
-    val onClosed: F[Unit],
-    val goAway: H2Error => F[Unit],
```

Comment:
Not sure this is pivotal, but since `H2Stream` is a package private we are breaking source compatibility (even though MiMa is okay with it).

Reply:
I am not sure source-compatibility has to be kept for package-private classes and methods. Is it a good practice from outside for others to add stuff to an existing package? I wish, though, we could make it a Java 9 module or something stronger.